HeterogeneousCore/ROCmServices/plugins/ROCmService.cc

```diff
@@ -51,6 +48,7 @@ class ROCmService : public ROCmInterface {
 };
 
 void setHipLimit(hipLimit_t limit, const char* name, size_t request) {
+#if HIP_VERSION >= 50400000
```

Comment:
cmsdist is already in ROCm 5.6, so this `#ifdef` seems unnecessary in `master`?

Reply:
Ok. I'd be curious to learn more details on the hurdles at some point.

So on LUMI you basically compiled everything with the system ROCm, or something?

Reply:
> Ok. I'd be curious to learn more details on the hurdles at some point.

One of the things we learned during the LUMI hackathon is that ROCm has a more nuanced approach to the compatibility between the kernel driver and userspace libraries.
The [User/Kernel-Space Support Matrix](https://rocm.docs.amd.com/en/latest/release/user_kernel_space_compat_matrix.html) mentions that

> ROCm™ provides forward and backward compatibility between the Kernel Fusion Driver (KFD) and its user space software for +/- 2 releases.

and gives a list of combination that have been tested and are officially supported.

An additional hurdle is that there doesn't seem to be any way the query "KFD" version used in the table.
For example:
  - on LUMI, the system installation is ROCm 5.2.3, so the KFD should be version 5.2.3; however `rocm-smi --showdriverversion` gives `5.16.9.22.20`
  - on patatrack03, the system installation is ROCm 5.7.1, so the KFD should be version 5.7.1; however `rocm-smi --showdriverversion` gives `6.1.5`